<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela Inicial</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnLoga">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Logar</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>89</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblNome">
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSobrenome">
       <property name="text">
        <string>Sobrenome:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblIdade">
       <property name="text">
        <string>Idade:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSenha">
       <property name="text">
        <string>Senha:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>144</width>
      <height>140</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="txtNome">
       <property name="text">
        <string extracomment="Primeiro Nome"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSobrenome">
       <property name="text">
        <string extracomment="Seu sobrenome"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtIdade">
       <property name="text">
        <string extracomment="Sua idade"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSenha">
       <property name="text">
        <string extracomment="Até 8 digitos"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Opções</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionTela_Cheia"/>
    <addaction name="actionMinimizar"/>
    <addaction name="actionMaximizar"/>
    <addaction name="actionFechar"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFechar">
   <property name="text">
    <string>Fechar</string>
   </property>
   <property name="toolTip">
    <string>Fechar</string>
   </property>
  </action>
  <action name="actionTela_Cheia">
   <property name="text">
    <string>Tela Cheia</string>
   </property>
  </action>
  <action name="actionMinimizar">
   <property name="text">
    <string>Minimizar</string>
   </property>
  </action>
  <action name="actionMaximizar">
   <property name="text">
    <string>Maximizar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
